src/components/DepositWidget/index.tsx

```diff
@@ -465,7 +469,8 @@ const BalancesDisplay: React.FC<BalanceDisplayProps> = ({
 const BalancesDisplayMemoed = React.memo(BalancesDisplay)
 
 const DepositWidget: React.FC = () => {
-  const { balances, error } = useTokenBalances()
+  const { ethBalance } = useEthBalances()
+  const { balances, error } = useBalances()
```

Comment:
In that case (splitting eth balance from token balances), I'd say keep the old name for the hook (`useTokenBalances`)

Reply:
Granted: 179949ada1f31b266c425a3928dd7092a882454a